create-filter/index.js

```diff
@@ -152,22 +164,40 @@ export function createFilter(client, Template, filter = {}, opts = {}) {
               })
               .then(async () => {
                 await Promise.all(checking)
+
                 if (!Template.remote && isLoading) {
                   isLoading = false
                   filterStore.setKey('isLoading', false)
                   endTask()
                   resolve()
-                }
+                } else if (Template.remote) {
+                  const subscribeSinceLatest = latestMeta !== undefined
+                      ? { ...subscribe, since: { id: latestMeta.id, time: latestMeta.time } }
+                      : subscribe
+                  await client
+                    .sync(subscribeSinceLatest)
+                    .then(() => {
```

Comment:
Code coverage is not 100%. Maybe we should move `then()` and `catch()` processing into a function to not dublicate the code?

```js
await processSubscribe(client.sync(subscribeSinceLatest))
```

```js
await processSubscribe(client.sync(subscribe))
```

Reply:
nice! also found why ci was falling and fixed it in 9b981f9

do you aim for 100% coverage for branches also?